test/integration/features/F001_Valid_Change_Events.feature

```diff
@@ -55,3 +55,41 @@ Feature: F001. Ensure valid change events are converted and sent to DOS
     When the Changed Event is sent for processing with "valid" api key
     Then the Changed Request is accepted by Dos
     And the Dentist changes with service type id is captured by Dos
+
+@complete @splunk
+  Scenario: F001S009. Dentist Hidden reports to Splunk
+    Given a Dentist Changed Event is valid
+    And the Changed Event has ODS Code "V00393a"
+    When the OrganisationStatus is defined as "Hidden"
+    And the Changed Event is sent for processing with "valid" api key
+    Then the Event Processor logs to splunk with report key "HIDDEN_OR_CLOSED"
+
```

Comment:
Can all the references to Splunk be removed as nowhere in these tests ever check splunk